packages/client/wallets/smart-wallet/src/types/Error.ts

```diff
@@ -1,57 +1,51 @@
-export class NotAuthorizedError extends Error {
-    code = "ERROR_NOT_AUTHORIZED";
-
-    constructor(message: string) {
+export const SmartWalletErrors = {
```

Comment:
To be consistent with our other SDKs

Reply:
Did we need a not-authorized thats just for smart wallets? Or is there a chance we can have a core error for that across all routes